### ISSUE TITLE
Possible HTTPS issue

### ISSUE THREAD

**Issue by Byter09** [kyuucr/add-link-to-qbittorrent-webui#3]
I set up my web ui and added my Let's Encrypt wildcard certificate to it.

All I get is `XMLHttpRequest error orccured.`

If you want to test it yourself, here's the url:
[qBittorrent v4.0.3 Web UI](https://pi.btr.zone:8080/)

The settings I used are the following:
```
Profile:             Default
qBittorrent address: https://pi.btr.zone:8080/
```

You also said you never tried public web ui stuff, so if you need access, message me privately please.

Edit: I should probably note, that this is running on a Banana Pi and I'm currently seeding stuff so maybe it was a timeout?

**Comment by kyuucr** [comment]
Thanks, I do have suspicion that it will break on HTTPS, but I was too lazy to set up the server. I will look into it later, meanwhile you can send me your server auth to my email: muh.iqbal.cr[at]gmail[dot]com

**Comment by Byter09** [comment]
The mail is out - coming from `K. Becker <k.becker[at]btr[dot]zone>`.

If there's anything I can do to help, you can message me here or on the given mail address.

**Comment by kyuucr** [comment]
It actually just needs an https permission on the manifest, fixed on 997248b

Go ahead and delete the temp account. Also I added an ubuntu iso on you server as a test, you can remove it. I cannot remove it while it still downloading because of the heavy network load.

**Comment by Byter09** [comment]
Happy to hear, that it was such an easy fix. And yes my Pi is also busy calculating stuff in the background so it has a hard time handling all that. I should set up a cluster 😄
